### ISSUE TITLE
Does this work with modules which has entrypoints?

### ISSUE THREAD

**Issue by falu2010** [brentvollebregt/auto-py-to-exe#8]

**Comment by brentvollebregt** [comment]
If you are meaning scripts that import many other scripts you made, then yes.
For example, if you had a structure:
- run.py
- uitils.py
- helper.py
And run.py was the main script (entry-point) which imported the other scripts files, the all you need to do is use pyinstaller on run.py and the other two scripts will be included.

**Comment by falu2010** [comment]
No.. This clearly explain what is entrypoint https://amir.rachum.com/blog/2017/07/28/python-entry-points/ .Entrypoint is a very common use case in CLIs.

**Comment by brentvollebregt** [comment]
Ah right, I see what you are asking now. This application does not handle how these sorts of things are implemented; everything done here is simply passed to pyinstaller.

I assume you have tried to do what you want to do but with a simple google search referencing pyinstaller I found [this](https://github.com/pyinstaller/pyinstaller/issues/305). It is still open and not much activity has occurred on it lately so I am not sure if it has been added. Might be good to go and look at some docs.
